rts/Lua/LuaConstEngine.cpp

```diff
@@ -61,5 +72,13 @@ bool LuaConstEngine::PushEntries(lua_State* L)
 		LuaPushNamedNumber(L, "maxPiecesPerModel", MAX_PIECES_PER_MODEL);
 	lua_rawset(L, -3);
 
+	LuaPushNamedNumber(L, "gameSpeed"      , GAME_SPEED);
+
+	lua_pushliteral(L, "textColorCodes");
+	lua_createtable(L, 0, 3);
+		LuaPushNamedChar(L, "Color"          , static_cast<char>(CglFont::ColorCodeIndicator)  );
+		LuaPushNamedChar(L, "ColorAndOutline", static_cast<char>(CglFont::ColorCodeIndicatorEx));
+		LuaPushNamedChar(L, "Reset"          , CglFont::ColorResetIndicator );
```

Comment:
Nit: perhaps cast this one as well for consistency

Reply:
Done